docs/build/isc/v1.0.0-rc.6/docs/how-tos/EVM/send-funds-from-L1-to-L2.md

```diff
@@ -0,0 +1,105 @@
+---
+description: How to fund your L2 account.
```

Comment:
Wrong description and wrong sidebar title 🤔

Reply:
Fixed in https://github.com/iotaledger/iota-wiki/pull/1415/commits/70576612a9366b177f4ff2dff9685094cc1cb0cc